### PR TITLE
Magister Presence

### PR DESCRIPTION
**Pull request by QkeleQ10**
I've written a presence for the most popular (yet the worst) student information system in the Netherlands, Magister 6. This is my first pull request for this repository, so please tell me if I did anything wrong 😬

### CODE REVIEW

File: websites/M/Magister/dist/metadata.json [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  },
+  "url": "magister.net",
+  "regExp": "([a-z0-9-]+[.])*magister[.]net[/]",
+  "description": {
+    "en": "Magister 6 is a popular student information system in the Netherlands.",
+    "nl": "Magister 6 is een populair leerlingvolgsysteem in Nederland."
+  },
+  "version": "1.0.0",
+  "service": "Magister",
+  "logo": "https://play-lh.googleusercontent.com/Kx_AuRPEK6VR9Lzsn4p2l3QEkV_BHjofKX8UoacTaBpl1_kUmO6VAUyASU51FervXw=s360-rw",
```
Comment (Bas950) [inline websites/M/Magister/dist/metadata.json:15]: upload to imgur

**Review by Bas950 — CHANGES_REQUESTED** [review]
upload proof of it working

File: websites/M/Magister/dist/metadata.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
   },
   "version": "1.0.0",
   "service": "Magister",
-  "logo": "https://play-lh.googleusercontent.com/Kx_AuRPEK6VR9Lzsn4p2l3QEkV_BHjofKX8UoacTaBpl1_kUmO6VAUyASU51FervXw=s360-rw",
+  "logo": "https://i.imgur.com/cdTvHTu.png",
```
Comment (Bas950) [inline websites/M/Magister/dist/metadata.json]: logo must at least be 512x512

**Review by Bas950 — CHANGES_REQUESTED** [review]

File: websites/M/Magister/dist/metadata.json (hunk 2 above)
Comment (Bas950) [reply websites/M/Magister/dist/metadata.json]: Also fix codacy errors

**Comment by QkeleQ10** [comment]
image
image
image
image

File: websites/M/Magister/dist/metadata.json [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+  "version": "1.0.0",
+  "service": "Magister",
+  "logo": "https://i.imgur.com/1TtM9A1.png",
+  "thumbnail": "https://i.imgur.com/TVjDmC4.png",
+  "color": "#1F97F9",
+  "tags": [
+    "leerlingvolgsysteem",
+    "lvs",
+    "magister",
+    "6",
```
Comment (ririxidev) [inline websites/M/Magister/dist/metadata.json:22]: 6? xd

Comment (QkeleQ10) [reply websites/M/Magister/dist/metadata.json:22]: yeah idk the program is officially called Magister 6

Comment (ririxidev) [reply websites/M/Magister/dist/metadata.json:22]: but it isnt 6, delete it fr pls

**Review by Bas950 — APPROVED** [review]

**Review by ririxidev — APPROVED** [review]

**Pull request merged**